### DISCUSSION TITLE
download failed

### DISCUSSION THREAD

**Discussion by kkoccengiz** [aclap-dev/video-downloadhelper#668] [Help and Q&A]
### Your question

{
  "version": "9.0.1.22",
  "target": "google",
  "channel": "stable",
  "page_url": "https://filmizlehdfilm.com/10028-ic-savas-full-izle",
  "error": {
    "details": "SideDownload error: : Not detecting m3u8/hls with non standard extension and non standard mime type\n    Last message repeated 7 times\n[in#1 @ 0x7fb645739fc0] Error opening input: Invalid data found when processing input\nError opening input file https://streammovie.club/m3/VWF6MDdoUXQrclNMaXQwdGxza0o1VXdMbTUzaFpFTWdvUklGSDhzSGdBWjdsaG1mVFlHYUF1eHJQUDNDamxOalVVSjNMSlA5WTZtc2hVZXNEN21IL3dZQ3VmTFZndzZFMzNuNk1FeDNXZjZnclovb2Qvbml5THZFR0tqSGNEZjlrcUpLdkJqME11WUk4QlZTeTh0VWQ0SytyY3F0WXpKS0ZkbGNyZWNOT29hV1BSSGZrN292TEZPWk4rc3BnTEdHak02ZXRFdE1sSnYyMDRjS3loWmp2OERtQThXOXF4MFVYMFpXQkc4cDNobXdHNWxkUDFJenpBZVhIYm9PUnlyczh5amc1c2s0UTlobzdZaERyYjVTZzR0TWM1QmJjV1YxQ3ZCay91aHkvODQ9.\nError opening input files: Invalid data found when processing input\n",
    "downloadable_id": "downloadable_338277688184566",
    "error": "coapp_failure",
    "id": "coapp_failure_787414a4-2ac4-4c74-957b-f175c1794b2a"
  }
}

### Addon & CoApp version and details

{
  "version": "9.0.1.22",
  "target": "google",
  "channel": "stable",
  "page_url": "https://filmizlehdfilm.com/10028-ic-savas-full-izle",
  "error": {
    "details": "SideDownload error: : Not detecting m3u8/hls with non standard extension and non standard mime type\n    Last message repeated 7 times\n[in#1 @ 0x7fb645739fc0] Error opening input: Invalid data found when processing input\nError opening input file https://streammovie.club/m3/VWF6MDdoUXQrclNMaXQwdGxza0o1VXdMbTUzaFpFTWdvUklGSDhzSGdBWjdsaG1mVFlHYUF1eHJQUDNDamxOalVVSjNMSlA5WTZtc2hVZXNEN21IL3dZQ3VmTFZndzZFMzNuNk1FeDNXZjZnclovb2Qvbml5THZFR0tqSGNEZjlrcUpLdkJqME11WUk4QlZTeTh0VWQ0SytyY3F0WXpKS0ZkbGNyZWNOT29hV1BSSGZrN292TEZPWk4rc3BnTEdHak02ZXRFdE1sSnYyMDRjS3loWmp2OERtQThXOXF4MFVYMFpXQkc4cDNobXdHNWxkUDFJenpBZVhIYm9PUnlyczh5amc1c2s0UTlobzdZaERyYjVTZzR0TWM1QmJjV1YxQ3ZCay91aHkvODQ9.\nError opening input files: Invalid data found when processing input\n",
    "downloadable_id": "downloadable_338277688184566",
    "error": "coapp_failure",
    "id": "coapp_failure_787414a4-2ac4-4c74-957b-f175c1794b2a"
  }
}

**Comment by Mattmjs** [comment]
I didn't get detection on Gold HD version, there was detection on VIP HD version. The download started and looks to work.
Although I don't feel like downloading something under 2 hours so I stop it around 5%.
This is on Windows 10 using Firefox, 9.0.1.13 and companion version 2.0.19. Just another user like you.

Your on Chrome but no other information supplied, also when did the error happen ?
At the start of download or end of download ?

**Reply by kkoccengiz** [reply]
I received a video helper update. After installing it, the plugin became unusable and the download never started.

**Reply by Mattmjs** [reply]
I did the same thing in ungoogled chromium with version 9.0.1.22, it started downloading until I quit at 5%.

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user.

I agree.  VDH can't detect the movie here.

As usual, I went to Plan B.  As I was trawling through the Network Monitor, my eye happened to land on the entry highlighted in this image:

![#01](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/6b9b7a07-4cfd-43fb-89a4-d772cdf6edf2)

I thought that looks suspiciously like an HLS master manifest masquerading as a plain text file.  I saved it on my system.  I ran it through ffprobe.  Here are the results:

[ffprobe.txt](https://github.com/aclap-dev/video-downloadhelper/files/15446763/ffprobe.txt)

So I executed ffmpeg using that information:

[Movie mp4 Log.txt](https://github.com/aclap-dev/video-downloadhelper/files/15446766/Movie.mp4.Log.txt)

The download speed was pretty slow so it took a while to download.  Both the download time & the download speed are noted near the end of that log file.  This is what the file looked like:

![#02](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/a662713d-df14-425b-b60a-e53e33f40614)

And here it is playing in VLC.

![#03](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/50909d4d-d765-4ff7-bec8-7f77c015ef60)

I didn't sit & watch this, just skimmed it, sampling it at intervals.  But I did get the whole movie, opening credits, closing credits, everything.  You can see that the video played just fine.  You'll have to take my word for it that the audio was just as fine.

My opinion is that VDH should never work on this site.  The administrators of the site have decided to use completely non-standard conventions for supplying this content.  Maybe they thought they were protecting their content from downloading.  Maybe they're just incompetent.  Either way, VDH cannot possibly be expected to handle this case properly.  But my test proves you can easily get this content if you're willing to expand your horizons & learn something new.  First, you need to learn how to inspect the Network Monitor for clues, which may be quite hard to find.  But what I've done here shows that persistence can pay off.  Then you need to learn how to use ffmpeg.  I wrote a tutorial on ffmpeg.  You can find it by doing a Google search using "vdh ffmpeg" as your search key.  The first search result of that search will be a link to my tutorial.

So it's your choice.  You can give up hoping VDH will ever work on this site & give up hoping to download this content with VDH.  Or you can learn something new & download any of this sort of content, easily, whenever you want.  It's up to you.

**Comment by Mattmjs** [comment]
Willy if you read carefully read my first response then you'll know VDH
does work for this site. There are different video players.
The first doesn't detect anything but next option does work.
